tests/smoke/notion_install.rs

```diff
@@ -21,6 +21,18 @@ fn install_node() {
     p.assert_node_version_is_installed("10.2.1", "5.6.0");
 }
 
+#[test]
+fn install_node_lts() {
+    let p = temp_project().build();
+
+    assert_that!(p.notion("install node lts"), execs().with_status(0));
+
+    assert_that!(
+        p.node("--version"),
+        execs().with_status(0).with_stdout_contains("1.2.1")
```

Comment:
Since the LTS version of Node will constantly change, we probably can't check for a specific version here. However, checking that both the `notion install node lts` command completed successfully and then that `node --version` completes successfully should be enough of a smoke test that the install is working.

Reply:
😅 absolutely. I was putting a bad version in there so I could actually test that my changes were doing something but ended up getting a totally different error! Thanks for the suggestion on how to test this though 👍